<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="CXAnimateWidgetRoate" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>42</width>
        <height>42</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="CXSVGPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="CXAnimateWidgetMove" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>351</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CXAnimateWidgetRoate" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>10000</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="CXSVGPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CXSVGLabel" name="label_2">
         <property name="text">
          <string>/Users/edz/Downloads/new.svg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="CXSVGLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>323</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>/Users/edz/Downloads/new.svg</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CXSVGLabel</class>
   <extends>QLabel</extends>
   <header location="global">XSVGLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CXAnimateWidgetRoate</class>
   <extends>QWidget</extends>
   <header location="global">XAnimateWidget-Roate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CXSVGPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">XSVGPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CXAnimateWidgetMove</class>
   <extends>QWidget</extends>
   <header location="global">XAnimateWidget-Move.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
